static/src/stylesheets/inline/article-photo-essay.scss

```diff
@@ -0,0 +1,156 @@
+@import '../_vars';
+@import '../_mixins';
+
+.content--article {
+    a.u-underline.in-body-link--immersive {
+      color: $news-main-1;
+
+      &:hover {
+        border-bottom: 1px solid $news-main-2;
+      }
+    }
+
+    blockquote.photo-essay-block-quote {
+      font-family: $f-serif-headline;
+      font-style: normal;
+      margin: 0 0 $gs-baseline * 2;
+
+      &:not(.quoted) {
+        background-color: #f6f6f6;
+        padding: $gs-baseline / 1.5  $gs-baseline $gs-baseline;
+      }
+
+      h2 {
+        color: $news-main-1;
+        font-weight: 900;
+        font-size: 1.8rem;
+        line-height: 1.8rem;
```

Comment:
I was told, we're always writing font-sizes in `px`, since they are converted automatically. In this case you could also write `line-height: 1` I guess.